src/plugins/comments/commentEditor.js

```diff
@@ -152,11 +152,9 @@ class CommentEditor {
     const textArea = this.rootDocument.createElement('textarea');
     this.container = this.rootDocument.querySelector(`.${CommentEditor.CLASS_EDITOR_CONTAINER}`);
```

Comment:
```suggestion
```

I think this line is useless now

Reply:
Done https://github.com/handsontable/handsontable/pull/7579/commits/137f58458b0b21af2dada6ee077838b61fd0bea4